index.bs

```diff
@@ -855,28 +883,29 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
     <dl class="switch">
```

Comment:
[new line 881](https://github.com/w3c/webauthn/pull/498/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR881) above should be a substep of the step above it at [new line 878](https://github.com/w3c/webauthn/pull/498/files#diff-ec9cfa5f3f35ec1f84feb2e59686c34dR878). If others agree, will make that change in a subsequent commit.